.circleci/config.yml

```diff
@@ -131,6 +131,13 @@ jobs:
           command: |
             bash -i -c 'npm run circle-ci-license-test-file'
             bash scripts/detect-package-json-changes.sh
+      # Override the webservice version in package.json to instead build against develop webservice
+      - when:
+         condition: << pipeline.parameters.sanity-check-against-develop >>
+         steps:
+          - run:
+              name: set webservice to develop
+              command: bash -i -c 'npm config set dockstore-ui2:webservice_version develop'
```

Comment:
To be safe, I think you should also set the `use_circle` boolean in the package.json to `false` otherwise it will run against the CircleCI artifact defined in the package.json regardless of the `webservice_version`. https://github.com/dockstore/dockstore-ui2/blob/98d079ab6877602b5fb4c2e6133d95c44b4f430e/scripts/generate-openapi-script.sh#L17